src/test/java/net/snowflake/client/core/SessionUtilLatestIT.java

```diff
@@ -403,6 +403,7 @@ private SFLoginInput createOktaLoginInput() {
     input.setLoginTimeout(1000);
     input.setSessionParameters(new HashMap<>());
     input.setAuthenticator("https://testauth.okta.com");
+    input.setDisableSamlUrlCheck(false);
```

Comment:
The value is false by default. Is there any testing we can add for when it's true?

Reply:
Done.